cli/src/utils.rs

```diff
@@ -1107,7 +1108,7 @@ pub async fn deploy_payout_batch<S: Middleware + 'static>(
         Ok(gas) => gas,
         Err(error) => {
             let date = chrono::offset::Utc::now().to_string();
-            let file_path = PathBuf::from(&format!("./FailedPayouts{}", date));
+            let file_path = PathBuf::from(&format!("{}{}", FAILURE_FILE, date));
```

Comment:
this also seems like a real failure - if gas couldn't be estimated I think we would not send the transaction? probably a good idea for this to have a bespoke name too - maybe `GasEstimateFailedPayouts` ?